### PR TITLE
Fix for holidayId/eventId

### PR DESCRIPTION
**Pull request by paulbrownlee**
This pull request changes the holidayId to eventId to keep consistency .

### CODE REVIEW

File: web-app/src/utilities/AxiosInstance.js [hunk 1]
```diff
@@ -2,8 +2,8 @@ import axios from 'axios';
 import moment from 'moment';
 import store from '../store';
 
-const baseURL = process.env.DOMAIN;
-
+//const baseURL = process.env.DOMAIN;
+const baseURL = 'http://localhost:8081';
```
Comment (BrownMark) [inline web-app/src/utilities/AxiosInstance.js]: You'll need to undo this. In future create a .env file with DOMAIN set up.

Comment (paulbrownlee) [reply web-app/src/utilities/AxiosInstance.js]: Just pushed up the fixed AxiosInstance

**Comment by BrownMark** [comment]
What's the reason behind this PR? I would've though from the holiday controller we'd be specifying holiday IDs?

Is it to help out the FE so that they can use eventID everywhere?

**Comment by paulbrownlee** [comment]
William requested it in order for the front-end to keep the eventID consistent.

**Comment by William-Unosquare** [comment]
@BrownMark Yea this helps the front-end for that reason. I'd mentioned it in the internal chat on Slack a few days back. If this makes the back-end horrible though, we can still work with holidayId

**Review by eoinmcafee — APPROVED** [review]

**Review by eoinmcafee — APPROVED** [review]

**Review by eoinmcafee — APPROVED** [review]

**Pull request merged**